### PR TITLE
fix(map): fix the loading and centering of the map on the farm routes

### PR DESCRIPTION
**Pull request by markomalis**

### CODE REVIEW

File: src/client/pages/farm.vue [hunk 1]
```diff
@@ -25,7 +31,8 @@ export default {
   components: { FarmNav },
   data() {
     return {
-      contentIsOpen: false
+      contentIsOpen: false,
```
Comment (jbmoelker) [inline src/client/pages/farm.vue]: `contentIsOpen` -> `panelIsOpen`?
`panelContentLoaded` -> `contentIsLoaded`?

File: src/client/pages/farm.vue [hunk 2]
```diff
@@ -1,13 +1,19 @@
 <template>
-  <nuxt-child v-if="isIndex" />
+  <nuxt-child
+    v-if="isIndex"
+    @fitFeatures="fitFeatures"
+  />
   <div
     v-else
-    :class="{ [`farm-page__panel--wide`]: contentIsOpen }"
+    :class="{ [`farm-page__panel--visible`]: panelContentLoaded, [`farm-page__panel--wide`]: contentIsOpen }"
```
Comment (jbmoelker) [inline src/client/pages/farm.vue]:
```vue
<div :class="{ 
  [`farm-page__panel--visible`]: panelContentLoaded, 
  [`farm-page__panel--wide`]: contentIsOpen 
}">
```
-->
```vue
<div :class="{ 
  'farm-page__panel--visible': panelContentLoaded, 
  'farm-page__panel--wide': contentIsOpen,
}">
```

File: src/client/lib/mixins/require-features.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export default {
+  beforeCreate() {
+    if(process.server) {
+      return
+    }
+
+    const features = this.$store.state.mapbox.features
+    // If the user had not selected parcels redirect to the select parcels view
+    if (!features || !features.features.length) {
+      this.$router.replace('/farm')
```
Comment (jbmoelker) [inline src/client/lib/mixins/require-features.js]: `/farm` -> `/farm/`

File: src/client/pages/farm.vue [hunk 4]
```diff
@@ -47,14 +62,19 @@ export default {
 
 .farm-page__panel {
   position: relative;
-  min-width: 50%;
+  width: 0;
   height: 100%;
   overflow-y: auto;
   background-color: var(--background-light);
+  transition: .5s width cubic-bezier(0.075, 0.82, 0.165, 1);
```
Comment (jbmoelker) [inline src/client/pages/farm.vue]: Never animate width. Only use animate/transition on elements that only [trigger layout](https://csstriggers.com/) (not paint or composite).

You are better of animating in the panel from the left:
```css
.farm-page__panel {
  transform: translate(0);
  transition: .5s transform cubic-bezier(0.075, 0.82, 0.165, 1);
}
.farm-page__panel--loading {
  transform: translate(-100%);
}

File: src/client/store/mapbox/features/index.js [hunk 5]
```diff
@@ -26,16 +28,20 @@ export const actions = {
...
+    const map = rootGetters['mapbox/map']
+    const { features } = state
 
-      map.fitBounds(bounds, { zoom: 14 })
+    if(!features.length) {
+      return
     }
+
+    const bounds = geojsonExtent({
+      type: "FeatureCollection",
```
Comment (jbmoelker) [inline src/client/store/mapbox/features/index.js]: `"FeatureCollection"` -> `'FeatureCollection'`?

File: src/client/pages/farm/parcels.vue [hunk 6]
```diff
@@ -7,23 +7,28 @@
     >
       Percelen selecteren <md-icon>arrow_left_alt</md-icon>
     </md-button>
-    <parcels-table :parcels="features" />
+    <no-ssr>
+      <parcels-table :parcels="features" />
```
Comment (jbmoelker) [inline src/client/pages/farm/parcels.vue]: should we call them `parcels` everywhere instead of `features`?

Comment (markomalis) [reply src/client/pages/farm/parcels.vue]: i can do this in a seperate PR

**Pull request merged**